src/generators/azuredevops/generator/RepoGenerator.ts

```diff
@@ -40,9 +40,14 @@ export default class RepoGenerator implements IGenerator<GitRepository> {
       throw new Error('An error occurred while creating repository.');
     }
 
-    const orgName = repo.remoteUrl!.split('/')[2].split('.')[0];
-    const repoUrl = `https://${orgName}:${adoToken}@dev.azure.com/${orgName}/${encodeURIComponent(projectName)}/_git/${repoName}`;
-
+    var orgName : string = repo.remoteUrl!.split('/')[2].split('.')[0];
+    var repoUrl : string = "";
+    if(orgName != "dev")
+      repoUrl = `https://${orgName}:${adoToken}@dev.azure.com/${orgName}/${encodeURIComponent(projectName)}/_git/${repoName}`;
+    else
+      orgName = repo.remoteUrl!.split('/')[3].split('.')[0];
+      repoUrl = `https://${orgName}:${adoToken}@dev.azure.com/${orgName}/${encodeURIComponent(projectName)}/_git/${repoName}`;
+   
```

Comment:
```suggestion
    const orgName : string = repo.remoteUrl!.startsWith('https://dev.azure.com/')
        ? repo.remoteUrl!.split('/')[3] // for https://dev.azure.com/{orgname}/
        : repo.remoteUrl!.split('/')[2].split('.')[0]; // for https://{orgname}.visualstudio.com/
    const repoUrl = `https://${orgName}:${adoToken}@dev.azure.com/${orgName}/${encodeURIComponent(projectName)}/_git/${repoName}`;   
```

I think this is a little simpler and resolves the linter issues.